### ISSUE TITLE
I changed the input picture, the output became incorrect!

### ISSUE THREAD

**Issue by nongqiqin** [geyuying/PF-AFN#22]
Hello, excuse me!
I use my own pictures, the effect is very **poor**
![image](https://user-images.githubusercontent.com/31816449/114539141-c2a3d680-9c86-11eb-9aa5-25e1350f7652.png)
But with sample pictures, the effect will become very **good**!
![image](https://user-images.githubusercontent.com/31816449/114540748-7e193a80-9c88-11eb-8892-972a4be5acfc.png)
I have shrunk this picture to the same size as the case picture. Same as the example below:
![image](https://user-images.githubusercontent.com/31816449/114540452-1cf16700-9c88-11eb-8c17-dc1e72ccc6e5.png)
Can you give me a solution.
Thank you very much!

**Comment by geyuying** [comment]
Hi, the provided model is trained on VITON training set and evaluated on VITON test set. If you want to use your own picture, you need to train your own model.

**Comment by nongqiqin** [comment]
> Hi, the provided model is trained on VITON training set and evaluated on VITON test set. If you want to use your own picture, you need to train your own model.

Thank you so much for replying to me!
I think it is the problem of warp_model, I want to find a way to warpe the clothes better, and then input warped_edge into gen_model, will the effect be better?

**Comment by JohnAILove** [comment]
> > Hi, the provided model is trained on VITON training set and evaluated on VITON test set. If you want to use your own picture, you need to train your own model.
> 
> Thank you so much for replying to me!
> I think it is the problem of warp_model, I want to find a way to warpe the clothes better, and then input warped_edge into gen_model, will the effect be better?

我也有一样的问题，可以+WX讨论不??
a89879896

**Comment by Testhjf** [comment]
@nongqiqin  canyou share the checkpoints model
